src/ntfs/read.rs

```diff
@@ -97,8 +98,16 @@ pub fn manage_stops(collections: &mut Collections, path: &path::Path) -> Result<
                 }
                 stop_points.push(StopPoint::from_with_type(stop, stop_type));
             }
-            1 => stop_areas.push(StopArea::from(stop)),
-            i => warn!("stop.location_type = {} not yet supported, skipping.", i),
+            StopLocationType::StopArea => stop_areas.push(StopArea::from(stop)),
+            StopLocationType::EntranceExit => {
```

Comment:
I was wondering if something _like_ the following (the code below doesn't build) was possible but not really sure how and it's just a curiosity (I'm fine if we merge as it is).

```
stop_location_type => warn!(
  "stop.location_type = {} not yet supported, skipping.", 
  stop_location_type.serialize(?serializer?)
),
```